tests/Tests.Core/Client/Settings/TestConnectionSettings.cs

```diff
@@ -55,6 +55,9 @@ public class TestConnectionSettings : ConnectionSettings
 			.ConnectionLimit(ConnectionLimitDefault)
 			.OnRequestCompleted(r =>
 			{
+				//r.HttpMethod;
```

Comment:
In by accident?

Reply:
in by accident will remove thanks for catching this!